python/Ice/filesystem/client/main.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# Copyright (c) ZeroC, Inc.
+
+import Ice
+import asyncio
+import sys
+
+# Slice module Filesystem in Filesystem.ice maps to C# namespace Filesystem.
+from Filesystem import DirectoryPrx, FilePrx
+
+async def main():
+    # Create an Ice communicator to initialize the Ice runtime. It will be automatically destroyed at the end of the
+    # with block. We enable asyncio support by passing the current event loop to initialize.
+    with Ice.initialize(sys.argv, eventLoop=asyncio.get_running_loop()) as communicator:
+
+        # Create a proxy for the root directory.
+        rootDir = DirectoryPrx(communicator, "RootDir:tcp -h localhost -p 4061")
+
+        async def list_recursive(directory, depth=0):
```

Comment:
Can we move `list_recurvise` elsewhere?

I find it hard to follow to have this big local function in the middleware of the code.

Reply:
Moved the definition out before main.